packages/react-components/src/components/SendBox.tsx

```diff
@@ -305,18 +305,18 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   }, [props.activeFileUploads, fileUploadsPendingError]);
 
   /* @conditional-compile-remove(file-sharing) */
-  const onRenderFileUploads = useCallback(
-    () =>
-      props.onRenderFileUploads ? (
-        props.onRenderFileUploads()
-      ) : (
-        <_FileUploadCards
-          activeFileUploads={props.activeFileUploads ? props.activeFileUploads : []}
-          onCancelFileUpload={props.onCancelFileUpload}
-        />
-      ),
-    [props]
-  );
+  const onRenderFileUploads = useCallback(() => {
+    if (props.activeFileUploads?.length === 0) {
+      return null;
+    }
+    return props.onRenderFileUploads ? (
+      props.onRenderFileUploads()
+    ) : (
+      <div style={{ margin: '0.25rem' }}>
```

Comment:
From the snapshot of the sendbox, this looks to always be being applied even when no fileUploads are present?
![image](https://user-images.githubusercontent.com/2684369/161865541-0160778c-dedd-4a38-a179-34e4be413099.png)

This looks to be causing the sendbox being slightly too big when used just as a component (outside of a composite):
![image](https://user-images.githubusercontent.com/2684369/161865657-2b40558a-b82c-4d22-a415-986f474c3f79.png)

Reply:
Yes, pushed another commit to fix this. :)

Reply:
Fixed